doc/self_hosted_ci_config.md

```diff
@@ -6,7 +6,7 @@ machine.
 ## Configuration
 
 ```bash
-sudo apt-get install libssl-dev zsh build-essential pkg-config git gcc clang libclang-dev python3-pip hub numactl
+sudo apt-get install libssl-dev zsh build-essential pkg-config git gcc clang libclang-dev python3-pip hub numactl cmake
```

Comment:
why?

Reply:
Leonid says it's needed by kafka (https://github.com/vmware/database-stream-processor/pull/240)